backmerge 10-02-25

technophile-04 opened the pull request:
- **fix typo in package.json scripts for lint** (https://github.com/scaffold-eth/scaffold-eth-2/pull/1041)
- **Add `hardhat:clean` command to root of monorepo (https://github.com/scaffold-eth/scaffold-eth-2/pull/1043)**
- **Only render FaucetButton on localhost (https://github.com/scaffold-eth/scaffold-eth-2/pull/1047)**
- **fix: remove indexed event params (https://github.com/scaffold-eth/scaffold-eth-2/pull/1048)**

technophile-04 commented:
Note we should do a normal merge commit

rin-st commented:
Hey! 

Wagmi/viem downgrade and sort order already backmerged https://github.com/scaffold-eth/create-eth/pull/206
Also, there are no changes in PR for lint typo. Updated it for hardhat [8b2104a](https://github.com/scaffold-eth/create-eth/pull/211/commits/8b2104a10fea6f71cebe087fd554da74b5041abc). 

See also https://github.com/scaffold-eth/create-eth/pull/212 . Moved it out of backmerge since it's foundry changes

technophile-04 commented:
> Wagmi/viem downgrade and sort order already backmerged https://github.com/scaffold-eth/create-eth/pull/206

Ohh actually #206 was merged with squash & merge :( and that's why it got include it this as well. 

Just updated the changeset so that we don't mention it in changelog.md 

> Also, there are no changes in PR for lint typo. Updated it for hardhat [8b2104a](https://github.com/scaffold-eth/create-eth/pull/211/commits/8b2104a10fea6f71cebe087fd554da74b5041abc).

Nice catch! Thanks Rinat!

rin-st reviewed (approved):
Lgtm!
